pkg/executor/common.go

```diff
@@ -181,14 +184,14 @@ func GetJobPods(podsClient tcorev1.PodInterface, jobName string, retryNr, retryC
 	}
 	if len(pods.Items) == 0 {
 		time.Sleep(time.Duration(retryNr * 500 * int(time.Millisecond))) // increase backoff timeout
-		return GetJobPods(podsClient, jobName, retryNr+1, retryCount)
+		return GetJobPods(ctx, podsClient, jobName, retryNr+1, retryCount)
 	}
 	return pods, nil
 }
 
 // GetPodLogs returns pod logs bytes
-func GetPodLogs(c kubernetes.Interface, namespace string, pod corev1.Pod, logLinesCount ...int64) (logs []byte, err error) {
-	count := int64(100)
+func GetPodLogs(ctx context.Context, c kubernetes.Interface, namespace string, pod corev1.Pod, logLinesCount ...int64) (logs []byte, err error) {
+	var count int64 = 100
```

Comment:
Is 100  a const?

Reply:
I don't know, this was from before, I just changed the semantics. Do you maybe know?

Reply:
No, I didn't see it before. Put 100 to const at the beginning of the file